paper.tex

```diff
@@ -1360,81 +1360,48 @@ \section*{ARG inference methods}\label{ARG_inference_methods}
+can be used on megabase scale data and hundreds of thousands of samples under human-like parameters.
+
 % AI: just to note, these papers do not all agree in their definition of an ARG.
 % I guess this is one of the points of the present paper, but should we mention this fact?
 % Should we also mention that most of these infer just the topology, but some also the times?
 
-
-Espalier~\citep{rasmussen2022espalier} uses Maximum Agreement Forests to
-minimise the differences between local trees estimated using other methods.
-
-\subsection*{Model-based methods}
-An alternative approach is to treat the ancestry as a latent parameter to be averaged out
+An alternative approach is to treat the ARG as a latent parameter to be averaged out
 by Monte Carlo methods, based either on importance sampling
 \citep{griffiths1996ancestral, fearnhead2001estimating, jenkins2011inference}
 or MCMC \citep{kuhner2000maximum, nielsen2000estimation, wang2008bayesian, fallon2013acg}.
-These methods invariably operated on a representation of the ``little ARG'', typically
-generated until the grand MRCA. They are extremely computationally expensive,
-and applicable to at most hundreds of samples consisting of tens of kilobases with
-human-like parameters.
-
-Due to the computational expense of sampling ``little ARGs'', state-of-the-art
-Monte Carlo methods typically rely on cheaper, approximate models.
-The most prominent examples are
-\texttt{ARGWeaver}~\citep{rasmussen2014genome,hubisz2020mapping,hubisz2020inference}
-and \texttt{Arbores} \citep{heine2018bridging}. The former also utilises a time
-discretisation approximation, but scales to dozens of mammal-like genomes.
-The \texttt{ClonalOrigin} method \citep{didelot2010inference,
-medina2020speeding} uses the \texttt{ClonalFrame}
-model and data structure, though it is not as scalable as \texttt{ARGWeaver}.
-
-A very recent MCMC method called \texttt{ARGInfer} has resulted in the first
-improvement in the scalability of Monte Carlo methods for the exact ARG model
-by making use of the tree sequence representation of marginal trees, suitably enriched
-to facilitate the computations needed for MCMC sampling
-\citep{mahmoudi2022bayesian}.
-While not competitive with the scalability of \texttt{ARGWeaver}, \texttt{ARGInfer} extends
-the feasible range of sequence lengths by at least an order of magnitude when
-compared with methods which sample realisations of the exact ``little ARG''.
+These methods operate on representations of the ``little ARG'', and are extremely computationally
+expensive, being applicable to at most hundreds of samples consisting of tens of kilobases with
+human-like parameters. State-of-the-art methods rely on cheaper, approximate models
+\citep{didelot2010inference, rasmussen2014genome, heine2018bridging, hubisz2020mapping,hubisz2020inference, medina2020speeding}. 
+The most scalable method, \texttt{ARGWeaver}, can be applied to dozens of mammal-like
```

Comment:
Maybe better to delete the rasmussen2014genome reference from the previous sentence and put it just after "ARGweaver" (otherwise it's not clear what the reference for ARGweaver is)?